netcore/README.md

```diff
@@ -2,5 +2,5 @@
 1. clone repo
 1. navigate to /netcore
 1. use **generate-cert.sh** to generate a **server.pfx** certificate in the same folder - make sure to specify all passwords [link](https://razvangoga.blob.core.windows.net/share/http2-certificate-generation.png)
-1. execute **dotnet run {{cert_pass}}** - the webservice answers on 8080
+1. execute **dotnet run -c release {{cert_pass}}** - the webservice answers on 8080
```

Comment:
For the record this is what @razvangoga suggested at https://github.com/martin-g/http2-server-perf-tests/pull/4#issuecomment-706143730. But it didn't change much the results.